### DISCUSSION TITLE
Search a specific path

### DISCUSSION THREAD

**Discussion by kumar007git** [adirh3/Fluent-Search#680] [General]
I have a whole bunch of shortcuts to apps in a folder. How do I tell fluent search to look in that folder first when I type in the search box? thanks.

**Comment by adirh3** [comment]
Few options 
1. Use path search in files e.g. `myfolder\myfile`
2. Use full path as search tag, e.g. `C:\Apps` press TAB, search
3. Use search tag mapping for custom tag, in Settings -> Search tag, e.g. map `C:\Apps` to `myapps`, and use tag as tag
4. If you want these to show as apps, add the path in `Custom apps paths` in Settings -> Apps
